nextflow.config

```diff
@@ -31,8 +31,10 @@ params {
   meth_cutoff = false
   methyl_kit = false
   min_depth = 0
+  min_coverage = 1
   skip_deduplication = false
   non_directional = false
+  nondirectional_library = false
```

Comment:
the `bismark` workflow aslo has `params.non_directional`, could reuse that for `BISCUIT` aswell

Reply:
That's what I did in the beginning, but then I had a problem witth the `nextflow_schema.json` file, so I added this and `min_coverage` (instead of `min_depth`) for the BISCUIT aligner

Reply:
What was the problem? That the parameter can't be in the schema twice and so you have to choose between the Bismark and Biscuit groups?

If so then in other cases I have made a new group at the top called "Common" etc. for parameters like this.

Reply:
Yes, but there are two parameters: `params.non_directional` which is common between Bismark and BISCUIT, and `params.min_coverage` which is common between bwa-meth and BISCUIT.
Would you still do "Common" group for such a case?

Reply:
Done 👍 
Though there's a problem in the help message, I don't know how to solve it.
Copying here the "common option" group help message:
```
common options
  --min_depth                       [integer] Specify a minimum read coverage for MethylDackel to report a methylation call in bwa-meth workflow, or a minimum
                                              read coverage for information extraction from the VCF file to bed file in BISCUIT workflow.
  --non_directional                 [boolean] Run alignment against all four possible strands.

```

There's a whitespace between "minimum" and "read" and I don't know what causes this problem